cfgov/jinja2/v1/_includes/atoms/hyperlink.html

```diff
@@ -1 +1,30 @@
-<a href="{{ value.url }}">{{ value.text }}</a>
+{# ==========================================================================
+
+   hyperlink
+
+   ==========================================================================
+
+   Description:
+
+   Create a hyperlink atom when given:
+
+   value:         An object with the following options for value.
```

Comment:
Elsewhere we've been passing in two parameters: `settings` and `options`. The first is for required parameters, the latter is for optional parameters. Would that make sense on the django-side or are conventions for the parameter to always be `value`?